server/utilities/helperFunctions.ts

```diff
@@ -152,3 +152,122 @@ export function determineLoginId(auth) {
     const decodedToken = jwt.decode(auth, { json: true });
     return parseInt(decodedToken.sub.split('|')[1], 10);
 }
+
+export function timeToMilliSeconds(time: string) {
+    const [hourMin, period] = time.split(' ');
+    const [hour, min] = hourMin.split(':');
+    const convertedHour = hour === '12' ? 3600000 : 3600000 * parseInt(hour, 10);
```

Comment:
I think you mean this?
```suggestion
    const convertedHour = hour === '12' ? 0 : 3600000 * parseInt(hour, 10);
```

Reply:
i marked everything else as resolved. just this one thing